Plugin/TUFX/Addon/TexturesUnlimitedFXLoader.cs

```diff
@@ -38,10 +38,16 @@ public class TexturesUnlimitedFXLoader : MonoBehaviour
         public class Configuration
         {
             [Persistent] public string MainMenuProfile = "Default-MainMenu";
+            [Persistent] public string SpaceCenterSceneProfile = "Default-KSC";
+            [Persistent] public string EditorSceneProfile = "Default-Editor";
+            [Persistent] public string FlightSceneProfile = "Default-Flight";
+            [Persistent] public string MapSceneProfile = "Default-Tracking";
+            [Persistent] public string IVAProfile = "Default-Flight";
+            [Persistent] public string TrackingStationProfile = "Default-Tracking";
             [Persistent] public bool ShowToolbarButton = true;
         }
 
-		internal Configuration configuration = new Configuration();
+		public static readonly Configuration defaultConfiguration = new Configuration();
```

Comment:
Is there a reason this is now public?  Are you making changes to some other mod that uses this?

Reply:
Ah, I got `internal` and `protected` mixed up :sweat_smile: